### PR TITLE
Added the --no-headless option back in

### PR DESCRIPTION
**Pull request by paulb-elastic**
This capability has been added back in via https://github.com/elastic/synthetics/pull/813 so adding back to the docs

### CODE REVIEW

**Review by shahzad31 — APPROVED** [review]

File: docs/en/observability/synthetics-command-reference.asciidoc
Comment (vigneshshanmugam) [inline docs/en/observability/synthetics-command-reference.asciidoc:73]: Its good to inform users that we don't support running headful on the Synthetics global infrastructure. 

This is one of the main reason why I removed this flag as it was not uniform.

Comment (vigneshshanmugam) [reply docs/en/observability/synthetics-command-reference.asciidoc:73]: My worry is that, people start using it and not seeing results in their monitor. Not worried about CLI/PW Options, but just as a Note saying we dont support headful mode in our infrastructure.

Comment (paulb-elastic) [reply docs/en/observability/synthetics-command-reference.asciidoc:73]: @vigneshshanmugam good point, I've added a note to that effect if you could do another review

**Review by vigneshshanmugam — APPROVED** [review]
LGTM

File: docs/en/observability/synthetics-command-reference.asciidoc
Comment (colleenmcginnis) [reply docs/en/observability/synthetics-command-reference.asciidoc:73]: Do you think the audience for this doc will already be familiar with the terms "headless" and "headful"? If not, a link to the Playwright docs might be helpful. I'll leave it up to you!

```suggestion
This is the same as setting https://playwright.dev/docs/api/class-testoptions#test-options-headless[Playwright's `headless` option] to `false` by running `--playwright-options '{"headless": false}'`.
```

File: docs/en/observability/synthetics-command-reference.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/synthetics-command-reference.asciidoc:77]:
```suggestion
Headful mode should only be used locally to see the browser and interact with DOM elements directly for testing purposes. Do not attempt to run in headful mode when running through Elastic's global managed testing infrastructure or {private-location}s.
```

**Review by colleenmcginnis — APPROVED** [review]
Minor suggestions below. None are blocking!

File: docs/en/observability/synthetics-command-reference.asciidoc
Comment (vigneshshanmugam) [reply docs/en/observability/synthetics-command-reference.asciidoc:77]: Can we also mention that we dont support on our infrastructure instead of saying not to use it.

**Comment by paulb-elastic** [comment]
Thanks @colleenmcginnis and @vigneshshanmugam, I have applied your suggestions and will merge once the build finishes and I've checked the output.

**Pull request merged**
